indexer/altitude_loader.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+#include "indexer/feature_altitude.hpp"
+#include "indexer/index.hpp"
+
+#include "coding/memory_region.hpp"
+
+#include "std/unique_ptr.hpp"
+#include "std/vector.hpp"
+
+#include "3party/succinct/rs_bit_vector.hpp"
+
+namespace feature
+{
+class AltitudeLoader
+{
+public:
+  explicit AltitudeLoader(MwmValue const & mwmValue);
+
+  TAltitudes const & GetAltitudes(uint32_t featureId, size_t pointCount) const;
```

Comment:
On the one hand it's a cache declared as mutable. On the other hand if I remove the word mutable a lot of methods lost their constantness including:
FeaturesRoadGraph::GetRoadInfo
FeaturesRoadGraph::GetSpeedKMPH
FeaturesRoadGraph::FindClosestEdges
FeaturesRoadGraph::GetCachedRoadInfo
RoadGraph::GetOutgoingEdgesList
RoadGraph::GetIngoingEdgesList

Do you think it's worth removing mutable in that case?